tests/test_dse.py

```diff
@@ -625,7 +625,7 @@ def test_tti_rewrite_aggressive_wmpi():
 
 @switchconfig(profiling='advanced')
 @pytest.mark.parametrize('space_order,expected', [
-    (8, 150), (16, 266)
+    (8, 154), (16, 270)
```

Comment:
what, why?

Reply:
because `1/vp**2` needs to be computed now.